src/components/Explorer/queryTemplate.js

```diff
@@ -127,6 +130,9 @@ ${conjoin`matches.start_time >= extract(epoch from timestamp '${new Date(minDate
 ${conjoin`matches.start_time <= extract(epoch from timestamp '${new Date(maxDate.value).toISOString()}')`}
 ${conjoin`leagues.tier = '${tier}'`}
 ${isTi8Team ? 'AND teams.team_id IN (5, 15, 39, 67, 2163, 350190, 543897, 726228, 1375614, 1838315, 1883502, 2108395, 2586976, 5026801, 5027210, 5066616, 5228654, 5229127)' : ''}
+${megaWin ? 'AND ((matches.barracks_status_radiant = 0 AND matches.radiant_win) OR (matches.barracks_status_dire = 0 AND NOT matches.radiant_win))' : ''}
+${radiantMaxGoldAdvantage ? `AND (SELECT matches.radiant_gold_adv[array_upper(matches.radiant_gold_adv, 1)]) <= ${radiantMaxGoldAdvantage.value}` : ''}
+${radiantMinGoldAdvantage ? `AND (SELECT matches.radiant_gold_adv[array_upper(matches.radiant_gold_adv, 1)]) >= ${radiantMinGoldAdvantage.value}` : ''}
```

Comment:
do you need to do absolute value here so you pick up matches where radiant or dire were leading?

Reply:
This is getting too confusing. Do you want me to do Max Radiant Advantage / Min Radiant  Advantage and Max Dire Advantage / Min Dire Advantage?

So 4 new fields in total.

Reply:
I think this would also be the least confusing to the user.

Reply:
or we could use a range slider? https://codepen.io/davidchin/full/GpNvqw

Reply:
I think just max gold advantage is most useful to the user, but that means the query needs to use absolute values so it can pick up on both negative (dire leading) and positive (radiant leading) values.

My concern with splitting it into 4 fields is that the user can't combine them, e.g. if they want to look at games where a team led by 10K gold or more, they would need to make one query for the radiant and one for the dire.  If the query handles absolute values, then it can just look for matches where the last element of radiant_gold_adv was abs greater than 10K